### ISSUE TITLE
Tests still fail with threaded bleadperl

### ISSUE THREAD

**Issue by eserte** [doy/eval-closure#6]
The `t/close-over.t` test still fails with a threaded perl 5.21.10, installed PadWalker, and Eval-Closure-0.12. See http://www.cpantesters.org/cpan/report/71cce2da-d0b1-11e4-b27a-6988e0bfc7aa for a sample fail report. Maybe the fix in https://github.com/doy/eval-closure/issues/4 is not enough?

**Comment by jplesnik** [comment]
The failing test could be fixed be following change

```
diff -up Eval-Closure-0.12/t/close-over.t.orig Eval-Closure-0.12/t/close-over.t
--- Eval-Closure-0.12/t/close-over.t.orig       2015-05-07 10:59:29.593190089 +0200
+++ Eval-Closure-0.12/t/close-over.t    2015-05-07 10:59:56.121504379 +0200
@@ -44,7 +44,7 @@ use Test::Requires 'PadWalker';
         $b = $b->OUTSIDE;
     }
     my @visible_in_outer_scope
-        = grep { defined and $_ ne '&' }
+        = grep { defined and $_ ne '&' and $_ ne '' }
           map  { $_->PV }
           grep { $_->can('PV') }
           map  { $_->PADLIST->ARRAYelt(0)->ARRAY }
```

**Comment by doy** [comment]
This should be fixed in f76a19a.

**Comment by eserte** [comment]
Perfect, http://fast-matrix.cpantesters.org/?dist=Eval-Closure%200.13 looks good.
